### DISCUSSION TITLE
Accessing the ColumnSerializable

### DISCUSSION THREAD

**Discussion by eamnicoletti** [serverpod/serverpod#1167] [Q&A]
Is there any way to access a ColumnSerializable object field in a query, like this:

Conventional way:

class
```
class: Example
table: examples

fields:
  userId: int, parent=users
```

query
```
return await Example.find(
  session,
  where: (t) => t.userId.equals(id)
);
```

My issue (how could I compare the user `id` in a ColumnSerializable as below):
class
```
class: Example
table: examples

fields:
  user: User
```

query
```
return await Example.find(
  session,
  where: (t) => t.user.id.equals(id)
);
```

**Comment by vlidholt** [comment]
There is a `findById` method you can use.

**Reply by eamnicoletti** [reply]
But `findById` returns a single object. I want to return a list with a condition. Like:

`SELECT * FROM examples JOIN users ON users.id = $id;`

But, just as I showed in the second example class, the user field is a ColumnSerializable and is save as a json.

**Reply by vlidholt** [reply]
Is this not working?

```dart
return await Example.find(
  session,
  where: (t) => t.id.equals(id)
);
```
